### PR TITLE
[WIP] upgrade(@umijs/test): jest 26, jsdom 16

### PR DESCRIPTION
**Pull request by ycjcl868**
https://jestjs.io/blog/2020/05/05/jest-26

##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

**Review by PeachScript — APPROVED** [review]
+1

File: packages/test/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "^26.6.3",
     "jest-cli": "^26.6.3",
-    "jest-environment-jsdom": "^26.6.2",
+    "jest-environment-jsdom-fourteen": "1.0.1",
```
Comment (afc163) [inline packages/test/package.json:38]: jest-environment-jsdom 有啥问题？

Comment (ycjcl868) [reply packages/test/package.json]: 测试 ci broken 是不是 jsdom 问题

**Comment by ycjcl868** [comment]
"jest-environment-jsdom-14" 暂时还换不掉


image

**Comment by ycjcl868** [comment]
`TypeError: (0 , _escapeRegExp(...).default) is not a function`

升到 jest 26 ，babel 就会报这个错

**Pull request closed without merging**